Samples/VirtualCamera/VirtualCameraTest/PropertySheet.props

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros">
+    <OSROOT>K:\os.2020_2\public\</OSROOT>
+    <OSOBJ>K:\os.2020_2\obj\</OSOBJ>

```

Comment:
this is going to be public, we should defer to the SDK instead; we cannot assume an external user has OSROOT / OSOBJ access